src/main/java/org/jacksonlaboratory/model/entity/OntologyTermBuilder.java

```diff
@@ -55,6 +56,8 @@ public OntologyTermBuilder setTranslations(List<Translation> translations) {
 	}
 
 	public OntologyTerm createOntologyTerm() {
+		Objects.requireNonNull(id, "id for ontology term cannot be null!");
```

Comment:
If these two fields are non-null, you can increase the likelihood that the user will use the builder right by asking for the fields in the builder constructor.

So, instead of doing this:

```java
OntologyTermBuilder builder = new OntologyTermBuilder();

builder.setId(id).setName(name);

// possibly set other fileds and build at the end
OntologyTerm term = builder.createOntologyTerm();
```

you can do 

```java
OntologyTermBuilder builder = new OntologyTermBuilder(id, name);
// possibly set other fileds
OntologyTerm term = builder.setId(id).setName(name).createOntologyTerm()
```

This communicates that `id` and `name` are in fact required. You still have to do the null check, but this can make the user think.